Misc/NEWS.d/next/Library/2024-10-22-19-52-15.gh-issue-87063.fR-wJa.rst

```diff
@@ -0,0 +1,2 @@
+Allow to determine whether a :class:`multiprocessing.Process` has been closed
+via the :attr:`~multiprocessing.Process.closed` attribute.
```

Comment:
@rruuaanng: I encourage you strongly to read the devguide. This change is a **new feature**, so it needs a [What's New entry](https://devguide.python.org/core-developers/committing/#changes-that-require-what-s-new-in-python-entries). **_Please take a look at existing What's New entries for inspiration!_**

Reply:
I'm not sure if the new changes are satisfactory, but I read what's under Doc/whatnews :). Please review it again!

Reply:
No. I will repeat again: _please read the devguide_. If you don't, you will continue to waste review resources.

New features (and especially _any new API_, like this is) will need a What's New entry. I linked to the section in the devguide that talks about What's New entries, and I also asked you to look at existing entries for inspiration. Apparently, nothing of this was done.

I'll ask you again: please take a look at how we _word_ NEWS and What's New entries for _new APIs_, and adapt your PR to follow our conventions. Please listen to advice.